src/amo/pages/UserProfileEdit/index.js

```diff
@@ -463,8 +463,10 @@ export class UserProfileEditBase extends React.Component<Props, State> {
       userId,
     } = this.props;
 
+    let alternateOutput;
+
     if (!currentUser) {
-      return (
+      alternateOutput = (
```

Comment:
Since this is no longer returning, it would be safer to wrap everything below it in an else statement. I'm not sure what the error handling code is going to do, for example.